### PR TITLE
CIFAR-10 Estimator-based example for multi-device training

### PR DESCRIPTION
**Pull request by protoget**
Approved internally. Now a PR to merge into models to pair with incoming new TF performance tutorial.

### CODE REVIEW

**Comment by protoget** [comment]
@nealwu After talking w/ Toby and Martin, we'd wanna include this example asap. We wouldn't catch the garden release, but need this for the new TF perf tutorial. PTAL

File: tutorials/image/cifar10_estimator/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+CIFAR-10 is a common benchmark in machine learning for image recognition.
+
+http://www.cs.toronto.edu/~kriz/cifar.html
+
+Code in this directory focus on how to use TensorFlow Estimator to train and evaluate CIFAR-10ResetNet model, on a single host with one CPU and potentially multiple GPUs.
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/README.md]: "evaluate CIFAR-10ResetNet model" -> "evaluate a CIFAR-10 ResNet model"

File: tutorials/image/cifar10_estimator/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+<b>Prerequisite:</b>
+
+1. Install TensorFlow with GPU support  with a version more recent than 1.2.0-cp27.
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/README.md]: You can just say "Install TensorFlow version 1.2 or later with GPU support"

File: tutorials/image/cifar10_estimator/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+2. Download CIFAR-10 dataset.
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/README.md]: Download the

File: tutorials/image/cifar10_estimator/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+# Run the model on CPU and 2 CPUs. After training, it runs the evaluation.
+$ python cifar10_main.py --data_dir=/prefix/to/downloaded/data/cifar-10-batches-py \
+						 --model_dir=/tmp/resnet_model \
+						 --is_cpu_ps=False \
+						 --force_gpu_compatible=True \
+						 --num_gpus=2 \
+					     --train_steps=1000
+
+# There are more commandline flags to play with, check cifar10_main.py for details.
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/README.md]: commandline -> command line

Also, change the comma to a semicolon (run-on sentence)

File: tutorials/image/cifar10_estimator/README.md (hunk 3 above)
Comment (protoget) [inline tutorials/image/cifar10_estimator/README.md]: done

File: tutorials/image/cifar10_estimator/README.md (hunk 2 above)
Comment (protoget) [inline tutorials/image/cifar10_estimator/README.md]: 1.2.0 doesn't work -- has to be after cp27.

File: tutorials/image/cifar10_estimator/README.md (hunk 1 above)
Comment (protoget) [inline tutorials/image/cifar10_estimator/README.md]: done

File: tutorials/image/cifar10_estimator/README.md (hunk 4 above)
Comment (protoget) [inline tutorials/image/cifar10_estimator/README.md]: done

**Comment by protoget** [comment]
@nealwu I think Toby's working on a tutorial for performant models, so I guess it's under "tutorial" scope?

File: tutorials/image/cifar10_estimator/README.md (hunk 1 above)
Comment (nealwu) [inline tutorials/image/cifar10_estimator/README.md]: Looks like this didn't go through -- still says "ResetNet"

File: tutorials/image/cifar10_estimator/README.md (hunk 2 above)
Comment (nealwu) [inline tutorials/image/cifar10_estimator/README.md]: What do you mean? "cp27" just stands for CPU and Python 2.7.

**Pull request merged**